mods/d2k/rules/structures.yaml

```diff
@@ -800,6 +806,39 @@ palace:
 		DisplayRadarPing: True
 		CameraActor: camera
 		ActivationSequence:
+	ProduceActorPower@fremen:
+		Description: Recruit Fremen
+		LongDesc: Elite sniper infantry unit                       \n  Strong vs Infantry\n  Weak vs Vehicles\n  Special Ability: Invisibility
```

Comment:
Ah yes - this is a bit of a hack.  The spaces forces the dialog wider so that the timer in the tooltip doesn't collide with the name.